src/mscorlib/shared/System/ArgumentOutOfRangeException.cs

```diff
@@ -61,12 +63,13 @@ public ArgumentOutOfRangeException(String paramName, Object actualValue, String
         protected ArgumentOutOfRangeException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            _actualValue = info.GetValue("ActualValue", typeof(object));
         }
 
         public override void GetObjectData(SerializationInfo info, StreamingContext context)
         {
             base.GetObjectData(info, context);
+            info.AddValue("ActualValue", _actualValue, typeof(object));
```

Comment:
I've not looked at the reference source... I assume you went through and just used the code that was there, to ensure everything matches?

Reply:
Not reference source but I used what was there before Morgans change. I simply reverted (partly automatically) his change and put the typeforwards on it.

Reply:
I'm hand comparing with reference source.

Reply:
I am not checking though that all of these were originally in mscorlib (ie the type forward attribute should be mscorlib). Did you check that @ViktorHofer ?

Reply:
Yes, I did that by hand. That's the reason why it took me so long...